packages/cli/commands/project/upload.js

```diff
@@ -57,19 +57,22 @@ const uploadProjectFiles = async (accountId, projectName, filePath) => {
     )}`,
   });
 
+  let buildId;
+
   try {
     const upload = await uploadProject(accountId, projectName, filePath);
 
+    buildId = upload.buildId;
+
     spinnies.succeed('upload', {
       text: `Uploaded ${chalk.bold(projectName)} project files to ${chalk.bold(
         accountId
       )}`,
     });
 
     logger.debug(
-      `Project "${projectName}" uploaded and build #${upload.buildId} created`
+      `Project "${projectName}" uploaded and build #${buildId} created`
     );
-    await pollBuildStatus(accountId, projectName, upload.buildId);
```

Comment:
If this fails, the `spinnies.fail()` within the catch statement may throw an error because the spinner was already removed [above](https://github.com/HubSpot/hubspot-cli/pull/562/files/7706201da43b916d927b9ffbfb025b4b0364aaf8#diff-1989a521c805539a7ae5e60fa3f7b0f79d83106bbb225045cf33f831abf97757L63)